presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -603,11 +603,11 @@ public String toString()
 
     @ThreadSafe
```

Comment:
side note: you could split this commit to two or even three commits, one per each fix and for formatting